dali/pipeline/data/tensor_list.h

```diff
@@ -282,6 +285,27 @@ class DLL_PUBLIC TensorList : public Buffer<Backend> {
     return shape_;
   }
 
+  /**
+   * @brief Checks whether the TensorList is
+   * continuous. It returns true if and only if
+   * all of the stored Tensors are densely packed in memory.
+   */
+  inline bool IsContinuousTensor() const {
+    if (ntensor() == 0 || size_ == 0) {
+      return true;
+    }
+    Index offset = 0;
+
+    for (size_t i = 0; i < shape_.size(); ++i) {
+      const auto& o = shape_[i];
+      if (offset != offsets_[i]) {
+        return false;
+      }
+      offset += volume(o);
```

Comment:
```suggestion
      offset += volume(shape_[i]);
```
would read a little better, since you are only using `o` once

Reply:
Done